### PR TITLE
Add Datenanfragen, Saleor and SalesAgility hacktoberfest

### PR DESCRIPTION
**Pull request by karanrajpal14**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -368,6 +386,24 @@
     "dateAdded": "2019-10-14T16:16:36.000Z",
     "tags": ["clothing", "hacktoberfest"]
   },
+  {
+    "name": "Saleor",
+    "difficulty": "medium",
+    "description": "Solve any of the issues with the Hacktoberfest label and win a pair of Saleor Socks.",
```
Comment (aslafy-z) [inline data.json]: This should be an url of an image, it's a web page now.

Comment (karanrajpal14) [reply data.json]: Whoops! My bad. Missed that. I have fixed it now. Please have a look.

File: data.json [hunk 2]
```diff
@@ -359,6 +368,15 @@
     "dateAdded": "2019-10-12T19:49:00.000Z",
     "tags": ["hacktoberfest", "accessories"]
   },
+  {
+    "name": "Python Discord",
```
Comment (aslafy-z) [inline data.json]: Please drop this opportunity, see https://github.com/swapagarwal/swag-for-dev/pull/388#issuecomment-538707977

Comment (karanrajpal14) [reply data.json]: Done. I apologize for missing that as well.

**Comment by aslafy-z** [comment]
Thank you @karanrajpal14!

**Comment by karanrajpal14** [comment]
Happy to help!
Thank you for the opportunity to contribute @aslafy-z

**Pull request merged**